tests/test_select_country.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import time
+import requests
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+
+
+@pytest.mark.parametrize("file_format", ['id_formats_1', 'id_formats_2', 
+                                         'id_formats_3', 'id_formats_4', 
+                                         'id_formats_5'])
+def test_select_country(base_url, login, file_format, selenium, reload_until_condition):
+    selenium.get('{0}/'.format(base_url))
+
+    # go to country menu
+    country_page = selenium.find_element_by_link_text('➽ Existing excerpt / country')
+    country_page.click()
+
+    # select monaco 
+    country = selenium.find_element_by_xpath("//option[contains(.,'Monaco')]")
+    country.click()
+
+    # choose the file format
+    formats = selenium.find_element_by_id(file_format)
+    formats.click()
+
+    # submit
+    create = selenium.find_element_by_name('submit')
+    create.send_keys(Keys.RETURN)
+
+    # wait until the download link appears
+    btn_reload = selenium.find_element_by_link_text('↻ Reload')
```

Comment:
Please remove assignment, since only the side-effect of the `selenium.find_element_by_link_text('↻ Reload')` is needed here.